### DISCUSSION TITLE
Renovate uses wrong path to check for versions

### DISCUSSION THREAD

**Discussion by neilschark** [renovatebot/renovate#24957] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

latest docker image

### If you're self-hosting Renovate, select which platform you are using.

GitLab self-hosted

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Wanted end result.

Renovate should run without an error.

### What you tried so far.

Hello, I am not sure if this is a potential bug, or if I misconfigured renovate. I have a go project with many dependencies and renovate would run without a problem. Suddenly, renovate throws an error, as seen in the logs ('https://buf.build/gen/go/' not found\n")

The dependency in question is `buf.build/gen/go/bufbuild/protovalidate/protocolbuffers/go`. It seems that renovate is trimming this path and is only using `https://buf.build/gen/go/` to check for updates, but I have not configured it in any way. I tried updating renovate to the latest version, but no success.

### Relevant debug logs

<details><summary>Logs</summary>

```
"message": "remote: not found\nfatal: repository 'https://buf.build/gen/go/' not found\n",
             "stack": "Error: remote: not found\nfatal: repository 'https://buf.build/gen/go/' not found\n\n    at Object.action (/opt/containerbase/tools/renovate/37.3.0/node_modules/simple-git/src/lib/plugins/error-detection.plugin.ts:42:29)\n    at PluginStore.exec (/opt/containerbase/tools/renovate/37.3.0/node_modules/simple-git/src/lib/plugins/plugin-store.ts:29:29)\n    at /opt/containerbase/tools/renovate/37.3.0/node_modules/simple-git/src/lib/runners/git-executor-chain.ts:127:42\n    at new Promise (<anonymous>)\n    at GitExecutorChain.handleTaskData (/opt/containerbase/tools/renovate/37.3.0/node_modules/simple-git/src/lib/runners/git-executor-chain.ts:124:14)\n    at GitExecutorChain.<anonymous> (/opt/containerbase/tools/renovate/37.3.0/node_modules/simple-git/src/lib/runners/git-executor-chain.ts:100:40)\n    at Generator.next (<anonymous>)\n    at fulfilled (/opt/containerbase/tools/renovate/37.3.0/node_modules/simple-git/dist/cjs/index.js:55:24)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)"
```

</details>

**Comment by HonkingGoose** [comment, marked as answer]
This discussion seems related:

- #24639

**Reply by neilschark** [reply]
Thanks. Yes you are right, it seems to be the same problem.

I will look the discussion and check, if I find a solution there.

**Comment by rarkins** [comment]
https://github.com/renovatebot/renovate/pull/26746
